src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -193,6 +193,17 @@ def create_search_index(app, exception):
     excluded_docs = static_search_options.get("files_to_exclude", [])
     included_docs = app.env.found_docs
     filter_options = app.config.html_theme_options.get("search_filters", {})
+
+    # TODO (RV): Deprecate this in release 0.16.0
+    # https://github.com/ansys/ansys-sphinx-theme/issues/730
+    patterns = app.env.config.index_patterns or {}
+    if patterns:
+        logger.error(
+            "The 'index_patterns' is depreciated and no longer supported. "
+            "Please see the documentation https://sphinxdocs.ansys.com for the "
+            "new search configuration options."
+        )
```

Comment:
```suggestion
        error = ValueError(
            "The 'index_patterns' is deprecated and no longer supported. "
            "Please see the documentation https://sphinxdocs.ansys.com for the "
            "new search configuration options."
        )
        logger.error(error)
        raise error
```

Reply:
I am not able to add these suggestion since i pushed some change, may be at the same time you suggested, sorry for that, adapted this in https://github.com/ansys/ansys-sphinx-theme/pull/729/commits/1fedf2894ca5ca5bfd59ef020466a2b37692d5fd